modules/authglobus/lib/Auth/Source/Globus.php

```diff
@@ -116,6 +120,10 @@ private function getGlobusInfo($url, $token) {
 
         $context = stream_context_create($options);
         $info = file_get_contents($url, false, $context);
+        
+        if ($info === false) {
+            throw new SimpleSAML_Error_BadRequest('Could not retrieve identity information from Globus.');
```

Comment:
it would be useful to know why we could not get the identity information.
What was the return code?
What was the message?
Are there any other details we can log so that we dont have to modify the code to get the details back?